spring-boot-starter-camunda-connectors/src/main/java/io/camunda/connector/runtime/inbound/lifecycle/SpringInboundConnectorFactory.java

```diff
@@ -51,6 +52,9 @@ public InboundConnectorExecutable getInstance(String type) {
     if (webhookConnectorRegistry != null && WebhookConnectorRegistry.TYPE_WEBHOOK.equals(type)) {
       return new WebhookConnectorExecutable(webhookConnectorRegistry);
     }
+    if (webhookConnectorRegistry != null && "io.camunda:connector-sns-inbound:1".equals(type)) {
```

Comment:
Concern: we must have an indicator that a specific `@InboundConnector` is a webhook and has own context path.